classes/output/report_summary.php

```diff
@@ -140,7 +150,14 @@ public function export_for_template(renderer_base $output) {
         $this->validate_pricing();
 
         $context = new stdClass();
-        $context->total = '$' . number_format($this->calculate_total_cost(), 4);
+
+        $total = $this->calculate_total_cost();
+        if (!empty($total)) {
+            $context->total = '$' . number_format($total, 4);
```

Comment:
Does Moodle/PHP have a native format currency function?

Reply:
It has `money_format` but that requires having the correct locale, and I don't want to mess with that in case it affects language, and since all the prices are in USD, I don't want to rely on the set locale, as this may use a different symbol which is incorrect.